app/controllers/ops_controller.rb

```diff
@@ -787,7 +794,7 @@ def rebuild_toolbars(presenter)
 
   def handle_bottom_cell(nodetype, presenter, locals)
     # Handle bottom cell
-    if @pages || @in_a_form && locals[:action_url] != "rbac_tags_edit"
+    if @pages || @in_a_form && locals && locals[:action_url] != "rbac_tags_edit"
```

Comment:
Can we verify this didn't break any of the other paths in here like log_depot_edit or whatever ze is (zone edit?). log_depot_edit and ze don't use locals, so it's possible those paths never needed locals(?)